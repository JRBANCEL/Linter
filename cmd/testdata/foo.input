package foo

import (
	"fmt"
)

func bar() string {
	test := "baz"
	return fmt.Sprintf("YOLO: %v", test)
}
