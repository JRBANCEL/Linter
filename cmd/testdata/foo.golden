package foo

import (
	"fmt"
)

func bar() string {
	test := "baz"
	return fmt.Sprint("YOLO: ", test)
}
